### ISSUE TITLE
Low volume in video streaming in parallel with Twilio video on iOS 15

### ISSUE THREAD

**Issue by q-kit** [twilio/twilio-video.js#1785]
Please note that this is not the same with [another iOS 15 volume issue](https://github.com/twilio/twilio-video.js/issues/1586).

In my use case, the users are presenters. They publish their video and audio, but don't receive any video/audio from the subscribers. The presenter's UI has two videos side by side, one of which is their selfie video and the other of which is an instructional video streaming from some *.mp4 hosted in cloud.

It's been a while since the first time we saw the low volume issue on iOS 15, and it still repros on the latest iOS 15.5. Recently, we noticed that the low volume issue is related to the `createLocalAudioTrack` function. The instructional video sounds very soft (even in full volume bar) if `createLocalAudioTrack` is called beforehand, while it sounds much louder if no local audio track is enabled. Could you help with investigating? Thanks!

 - [x] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [x] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Software versions:**

 - [x] Browser(s): Mobile Safari
 - [x] Operating System: from iOS 15.0 to iOS 15.5
 - [x] twilio-video.js: 2.21.1
 - [x] Third-party libraries (e.g., Angular, React, etc.): vue.js, video.js (for playing the instructional video. It doesn't seem to be the cause of this issue as we can repro this issue with HTML video tag only)

**Comment by PikaJoyce** [comment]
Hi @q-kit,

Thanks for opening this issue. Do you have any logs/room sids for us to take a look to see if anything is wrong there? Or perhaps a repro link if possible in order for us to begin our investigations? 

Best,
Joyce

**Comment by q-kit** [comment]
Hello @PikaJoyce ,

Thanks for your reply!

I actually don't see any error/log in the dev tool, and I don't have any specific SID. Could you give my [app](https://app-staging.kit.com/user-test) a try with an iOS 15 device? You would notice that the audio in the volume check step and the instructional videos that follow would have very low volume. If you try quitting and reentering the app, the volume is much louder. You can also make the volume louder by restarting the `localAudioTrack` in the dev tool.

Please let me know how it goes and/or more info is needed from my end. Thanks!

Q
